service/awsconfig/v10/resource/s3bucket/error.go

```diff
@@ -22,21 +20,15 @@ func IsNotFound(err error) bool {
 	return microerror.Cause(err) == notFoundError
 }
 
+var noSuchBucketError = microerror.New("Not Found")
```

Comment:
You may want to read how we deal with errors. Please see https://github.com/giantswarm/fmt/blob/master/go/general_style_guide.md#errors. The matcher is `IsBucketNotFound`. The error should then be `bucketNotFoundError`. The message should then be `bucket not found`. Also, below you removed the string matcher checks. I did not request this. I am not into details here but this should may be reverted, or is there a specific reason for this?